app/main/posts/views/filters/active-search-filters.directive.js

```diff
@@ -151,8 +176,22 @@ function ActiveSearchFilters($translate, $filter, PostFilters, _, FilterTransfor
             });
         }
 
+        function removeCollection(collection, $event) {
+            $event.preventDefault();
+            $event.stopPropagation();
+            $scope.collection = null;
+            PostFilters.setMode('all', null);
+            /**
+             * because the default for collections is different than the default for
+             * other modes, we need to explicitly reset the filters or it will be shown
+             * as if we had manually set the collection status defaults [published, draft, archived]
+             */
+            PostFilters.setFilter('status', PostFilters.getDefaults().status);
+        }
+
+        // collections should not be saved within a saved search
         function showSaveSavedSearchButton() {
-            return !_.isEmpty($scope.uiFilters) && !$scope.savedSearch && $rootScope.loggedin;
+            return !$scope.collection && !_.isEmpty($scope.uiFilters) && !$scope.savedSearch && $rootScope.loggedin;
```

Comment:
as I understand it we shouldn't be able to save a saved search with a collection. Is that correct?